sdk/personalizer/Azure.AI.Personalizer/src/Models/RlObjectConverter.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
```

Comment:
Did you add unit tests for the converter and other classes you added which can be tested in isolation?

Reply:
Added unit tests for RlObjectConverter, DecisionContext, JsonRawStringListConverter. Please take a look when you have time.